docs/sphinx/conf.py.in

```diff
@@ -40,7 +40,7 @@ from datetime import datetime
 extensions = ['sphinx.ext.extlinks', 'edit_on_github']
 
 ## Configuration for the edit_on_github extension
-edit_on_github_project = 'openmicroscopy/bioformats'
+edit_on_github_project = 'ome/files'
```

Comment:
Probably.  This was likely from a blanket sed over the whole repo.  I can rename to ome/ome-files-cpp.

As a followup, does the repo need to be `ome-files-cpp` or could it just be `ome-files` and potentially have the java stuff merged in down the line?  I wasn't sure why we bothered with the suffix here; my understanding was that ome-common-cpp was a special-case due to the clash with java.